src/masonite/tests/TestCase.py

```diff
@@ -63,6 +65,9 @@ def setUp(self):
 
         self.routes_to_restore = set(self.application.make("router").routes)
 
+        # don't encrypt cookies during tests
```

Comment:
First solution would be disable cookies encryption during tests ? (done in last commit)

Reply:
But then CSRF token verif will fail... so that's not a good solution IMO.